packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SeparateFramesViewer/components/SeparateFrame/SeparateFrame.js

```diff
@@ -237,6 +232,7 @@ const SeparateFrame = ({
         }}
         direction='column'
         height='fit-content'
+        onKeyDown={onKeyDown}
```

Comment:
Noting that zooming with +/- keys doesn't work on Safari, but it's not related to these changes, it also doesn't work on other projects like https://www.zooniverse.org/projects/humphrydavy/davy-notebooks-project .

Reply:
Good to know!